Dockerfile.intel-nuc

```diff
@@ -0,0 +1,9 @@
+FROM resin/nuc-debian:jessie
+
+# Install the dependencies
+RUN apt-get update && apt-get install vim btrfs-tools mtools dosfstools python3 python3-requests python3-sh python3-parted python3-binaryornot jq -y && rm -rf /var/lib/apt/lists/*
+
+# Add the current directoy in the container
+ADD . /app
```

Comment:
This should probably be COPY, unless you're using any of the special features of ADD (eg automatic archive decompression)